### ISSUE TITLE
Feature Request: A way to tell when Simperium is done downloading data from the web on sign in.

### ISSUE THREAD

**Issue by lukegeiger** [Simperium/simperium-ios#404]
@jleandroperez As it sits right now, when a user logs back into my application, they probably will see no information because it is being downloaded from Simperium. It would be nice if I could have the ability to just show a loading indicator and wait on login before they were allowed to use the app.

**Comment by jleandroperez** [comment]
Closing in favor of Issue #442
